subquery/schema.graphql

```diff
@@ -65,9 +65,9 @@ type ERC721Token @entity @compositeIndexes(fields: [["owner", "contract"]]) {
   transfers: [ERC721Transfer!]! @derivedFrom(field: "token")
   application: String
   duration: Int
-  captureDate: BigInt @index
-  longitude: Float @index
-  latitude: Float @index
+  captureDate: BigInt
+  longitude: Float
+  latitude: Float
```

Comment:
I guess that's the sort of the changes that we need to reduce the unnecessary load from our indexers. Good job